pytest_django/fixtures.py

```diff
@@ -129,6 +132,16 @@ def handle_noargs(self, **options):
             patch_for_test_db_setup()
 
 
+def _disable_native_migrations():
+    from django import get_version
+
+    if get_version() > '1.7':
```

Comment:
Maybe it could be changed to return for `< '1.7'` instead, or try to import `DisableMigrations`.